### ISSUE TITLE
Duplicated roles

### ISSUE THREAD

**Issue by EgorBu** [bblfsh/python-driver#130]
Hi,
I found that roles are duplicated in many cases.

Ex: 
```
roles: FUNCTION
roles: CALL
roles: POSITIONAL
roles: ARGUMENT
roles: NAME
roles: FUNCTION
roles: CALL
roles: EXPRESSION
```

Code & extracted UAST

[bigartm.py](https://github.com/bblfsh/python-driver/files/1438275/bigartm.py.txt)
[bigartm_uast.txt](https://github.com/bblfsh/python-driver/files/1438268/bigartm_uast.txt)

**Comment by juanjux** [comment]
This is interesting... those are arguments that are also calls (like `foo(bar())`. So while I'll remove the first "CALL" (because it should be FUNCTION,ARGUMENT,ETC but not call for a single argument) it's legit that it also get the FUNCTION,CALL applied (since it's a function call) so we would end with 2 FUNCTION roles anyway.

The fix for that could be to implement [this feature request](https://github.com/bblfsh/sdk/issues/174) to remove duplicates already (we didn't implement it before because sometimes duplicated roles are a symptom of other bugs), but in this case the duplicate is unavoidable.

**Comment by juanjux** [comment]
Ok, so I'll:

- Remove the CALL role for function arguments (it'll be function+argument). This is what the Java driver does.
- Implement the SDK feature to uniq roles, since in this case there is no logical way to get a single FUNCTION role otherwise in this case.

Everybody ok with this?

**Comment by abeaumont** [comment]
@juanjux `java-driver` uses the `Call` role for function arguments.

**Comment by abeaumont** [comment]
Ok to the second point

**Comment by juanjux** [comment]
Ok, it should be a single CALL anyway with the uniq-fication.

**Comment by juanjux** [comment]
Update: I finished merged the SDK feature for removing duplicates, but for some reason the drivers are not showing this, even when rebuild against the new SDK (and with a server rebuild against it just to be sure), I'm investigating why this happen.

**Comment by jbeardly** [comment]
For sure is because you didn't created a tag

**Comment by juanjux** [comment]
I did (1.6.0).

**Comment by juanjux** [comment]
Will be definitely fixed as soon as bblfsh/sdk/pull/202 is merged and we tag new driver versions.

**Comment by juanjux** [comment]
Fixed in 1.2.2.
